api/events.go

```diff
@@ -36,15 +36,12 @@ func (eh *eventsHandler) Wait(remoteAddr string) {
 func (eh *eventsHandler) Handle(e *cluster.Event) error {
 	eh.RLock()
 
-	str := fmt.Sprintf("{%q:%q,%q:%q,%q:%q,%q:%d,%q:%q,%q:%q,%q:%q,%q:%q}",
+	str := fmt.Sprintf("{%q:%q,%q:%q,%q:%q,%q:%d,%q:%s}",
 		"status", e.Status,
 		"id", e.Id,
 		"from", e.From+" node:"+e.Node.Name(),
 		"time", e.Time,
-		"node_name", e.Node.Name(),
-		"node_id", e.Node.ID(),
-		"node_addr", e.Node.Addr(),
-		"node_ip", e.Node.IP())
+		"node", cluster.String(e.Node))
```

Comment:
I would probably call the function differently, such as `SerializeNode` or whatever.

`cluster.String` looks like it's going to Stringify a cluster, not a node